lib/statsd/instrument/batched_udp_sink.rb

```diff
@@ -54,130 +55,6 @@ def <<(datagram)
       def shutdown(*args)
         @dispatcher.shutdown(*args)
       end
-
-      class Buffer < SizedQueue
```

Comment:
Extracted into their own files so implementation can be shared outside of this class